ui/plugins/eu.esdihumboldt.hale.ui.application/plugin.xml

```diff
@@ -126,10 +126,6 @@
                name="aboutImage"
                value="images/hale_about.png">
          </property>
-         <property
```

Comment:
I'm not sure why Eclipse kicks that out when launching the product via the Product editor. Maybe it is no longer supported with Eclipse Mars, or it does not have to be stated because that's the default value anyway.